src/lib_c/x86_64-windows-msvc/c/int_safe.cr

```diff
@@ -1,3 +1,4 @@
 lib LibC
   alias DWORD = UInt32
+  alias DWORD_PTR = UInt32*
```

Comment:
I just used `DWORD*` instead of `DWORD_PTR` elsewhere, no need to copy *all* the aliases

Reply:
Removed this.